golang/cmd/factoryinsight/main.go

```diff
@@ -371,7 +371,7 @@ func setupRestAPI(accounts gin.Accounts, version int) {
 
 	*/
 	//dataFormat
-	err := router.Run(":80")
+	err := router.Run(":8080")
```

Comment:
I think you might have an accidential/test change in here.

Reply:
Hi, sorry that was an accident, I removed it from the commits.